src/About.js

```diff
@@ -40,6 +42,11 @@ export default function About() {
       <Row>
         {devdata}
       </Row>
+      <Row>
+        <Col className='mb-2'>
```

Comment:
Can you add COLs before and after the Footer COL here so it is consistent? This will mean adding responsive design by including sm={12} and md={12} (in my limited testing, both seem to be necessary). This should make it more consistent.